src/core/include/openvino/core/any.hpp

```diff
@@ -721,6 +721,28 @@ class OPENVINO_API Any {
         }
     }
 
+    template <class T,
```

Comment:
Revert changed from ov::Any

Reply:
This added template function `as_impl()` here is for loading the device utilization thresholds smoothly from JSON file via `benchamrk_app -load_config` . Should BA support to read utilization thresholds property from JSON file?

Reply:
The `core.compile_model()` Python binding also needs this added template function to convert from `ov::AnyMap(std::map<std::string, std::string)` to property type `ov::map<std::string, unsigned>`. 
https://github.com/openvinotoolkit/openvino/blob/1b1fa2de8d4306e350e9e609d8e75ca45f6c5107/src/bindings/python/src/pyopenvino/core/core.cpp#L158